tox.ini

```diff
@@ -30,6 +30,63 @@ deps =
 commands =
     python -m pytest --cov-report term --cov merlin -rxs tests/unit
 
+
+[testenv:test-nvtabular-cpu]
+passenv=GIT_COMMIT
+sitepackages=true
+allowlist_externals = git
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+commands =
+    ; the GIT_COMMIT env is the current commit of the core repo
+    ; NOTE!!!! We must clean this up in the jenkins configuration with `rm -rf "nvtabular-$GIT_COMMIT"`
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/NVTabular.git nvtabular-{env:GIT_COMMIT}
+    python -m pip install --upgrade "./nvtabular-{env:GIT_COMMIT}[cpu,dev]"
+    python -m pip install .
+
+    ; this runs the tests then removes the models repo directory whether the tests work or fail
+    python -m pytest nvtabular-{env:GIT_COMMIT}/tests/unit
+
+
+[testenv:test-models-cpu]
+passenv=GIT_COMMIT
+sitepackages=true
+allowlist_externals = git
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+commands =
+    ; the GIT_COMMIT env is the current commit of the core repo
+    ; NOTE!!!! We must clean this up in the jenkins configuration with `rm -rf "nvtabular-$GIT_COMMIT"`
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/models.git models-{env:GIT_COMMIT}
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/NVTabular.git nvtabular-{env:GIT_COMMIT}
```

Comment:
do we need to install nvtabular from a git clone here? (vs. pip `python -m pip install --upgrade nvtabular@git+https://github.com/NVIDIA-Merlin/NVTabular.git` )

Reply:
Yeah, we could do it that way too